src/Contract_Item.php

```diff
@@ -208,13 +208,18 @@ public function getTabNameForItem(CommonGLPI $item, $withtemplate = 0)
                 case Contract::class:
                     if ($_SESSION['glpishow_count_on_tabs']) {
                         $nb = self::countForMainItem($item);
+                        $nb += Contract_User::countForContract($item->fields['id']);
                     }
-                    return self::createTabEntry(_n('Item', 'Items', Session::getPluralNumber()), $nb, $item::class, 'ti ti-package');
+                    return self::createTabEntry(_n('Affected Item', 'Affected Items', Session::getPluralNumber()), $nb, $item::class, 'ti ti-package');
 
                 default:
                     if (
                         $_SESSION['glpishow_count_on_tabs']
-                        && in_array($item::class, $CFG_GLPI["contract_types"], true)
+                        && in_array(
+                            $item::class,
+                            array_merge($CFG_GLPI["contract_types"], [Contract_User::$itemtype_2]),
```

Comment:
It would be more meaningful for developers.

```suggestion
                            array_merge($CFG_GLPI["contract_types"], [User::class]),
```